pkg/ctl/topic/set_retention.go

```diff
@@ -58,7 +58,12 @@ func SetRetentionCmd(vc *cmdutils.VerbCmd) {
 		Out:  "[✖]  code: 404 reason: Namespace (tenant/namespace) does not exist",
 	}
 
-	out = append(out, successOut, noTopicName, tenantNotExistError, nsNotExistError)
+	topicLevelPolicyNotEnabledError := cmdutils.Output{
+		Desc: "topic-level policy is not enabled",
+		Out:  "[✖]  code: 405 reason: Topic level policy is disabled, please enable the topic level policy in brokers by configs of systemTopicEnabled and topicLevelPoliciesEnabled",
```

Comment:
Please address the style error

Reply:
I have to change the limit of line-length in golangci-lint to pass the ci...